config/prometheus/monitor.yaml

```diff
@@ -0,0 +1,26 @@
+
+# Prometheus Monitor Service (Metrics)
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
```

Comment:
Just curious, will this set up prometheus monitoring for the operator ?

Reply:
I can remove config/prometheus since we don't support it